doc/nrf/includes/sample_fem_support.txt

```diff
@@ -5,30 +5,17 @@
 You can add support for the nRF21540 front-end module to the sample.
 See :ref:`ug_radio_fem` for more information.
 
-To add support for the nRF21540 FEM, add the provided :file:`dts-nrf21540-fem.overlay` devicetree overlay file when building.
-The file is located in the |fem_file_path| folder.
```

Comment:
Given that `|fem_file_path|` is removed, it should be also removed from all applicable samples (e.g. Thread and Zigbee), as it is used to provide custom overlay source file path in each sample doc.

Reply:
Done